src/mococrw/symmetric_crypto.h

```diff
@@ -193,6 +193,17 @@ class AuthenticatedEncryptionI
      */
     virtual void setAuthTag(const std::vector<uint8_t> &tag) = 0;
 
+    /**
+     * Set associated data.
```

Comment:
The remaining parts of the crypto interface are a streaming API. To the best of my understanding this here is as well (as it might be called multiple times and will just add more data. So I think, we should rename it to addAssociatedData.

Reply:
True. I renamed it.